sites/hashdev/src/pages/_app.page.tsx

```diff
@@ -1,11 +1,12 @@
 /** @sync ../components/Snippet.tsx */
+import "../../styles/globals.css";
 import "../../styles/prism.css";
 
 import { EmotionCache } from "@emotion/react";
 import type { AppProps } from "next/app";
 import { useRouter } from "next/router";
 import { useEffect, FunctionComponent } from "react";
-import "../../styles/globals.css";
```

Comment:
Bonus: moved this import upwards where it belongs. When we switch to [eslint-plugin-simple-import-sort](https://github.com/lydell/eslint-plugin-simple-import-sort) this will happen anyway so it's better to do this in advance.